app/controllers/books_controller.rb

```diff
@@ -3,10 +3,12 @@ class BooksController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @books = Book.all
+    @books = Book.page(params[:page]).per(10)
```

Comment:
if you want to set constant on per page, you can do something like this, then you can remove per on this
```ruby
class Post < ActiveRecord::Base
  paginates_per 25
end
```